sdk/keyvault/tests.yml

```diff
@@ -17,6 +17,13 @@ stages:
           SubscriptionConfiguration: $(sub-config-cn-test-resources)
           MatrixFilters:
             - ArmTemplateParameters=^(?!.*enableHsm.*true)
+        Canary:
+          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+          Location: 'centraluseuap'
```

Comment:
Is this actually working for you currently, though? @benbp filed an IcM ticket to make sure the App Service docker container for attestation is working correctly in Canary. It's been failing ever since we switched to the "new" docker container method instead of a shared resource. Are you using that docker container? Your test-resources.json should look like .NET's if so, which currently fails in Canary (eastus2euap or centraluseuap - but must be centraluseuap to support linux containers).

Reply:
It is deploying successfully, so that's a mystery why NET is failing.

I believe my onboarding ticket was successful, so I'm curious if NET works now? If not, perhaps something else is different or mis-configured?

Reply:
As for `java - keyvault - tests-weekly` they haven't run in a long time. Still using the old shared resource: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1244531&view=logs&j=2c650e94-8dba-5eb5-ae19-43d8491a0830&t=1827f084-241e-5105-dc6a-23213a15e367&l=250

For this run, I see no evidence it ever tried to deploy to canary. Only region I'm seeing in any of the 4 matrix runs are in `westus`.

Reply:
I have not tried it yet, the live tests pipeline I ran only deploys in the public cloud. Now that you mention it, I should look into where these tests for other clouds are run...

Reply:
My mistake, I was looking at the wrong deploy job results.